<?xml version="1.0"?>
<launch>

    <node pkg="localizer" type="tf_publisher" name="tf_publisher" output="screen">	
		<remap from="/odom" to="/EKF/result" />
	</node>
	
	<node pkg="tf" type="static_transform_publisher" name="link3_broadcaster"
	  args="0 0 1.300 0 0 0 matching_base_link /velodyne 100" />

</launch>


